src/telegram.ts

```diff
@@ -282,7 +282,7 @@ class Telegram extends ApiClient {
    */
   sendVideo(
     chatId: number | string,
-    video: tt.InputFile,
+    video: tt.Opts<'sendVideo'>['video'],
```

Comment:
`typegram` makes sure that `string` is included in this definition. Videos can be sent via file_id.